src/flows/parser_item_html_text_streaming_flow.py

```diff
@@ -133,26 +197,82 @@ def load(key: str):
     uri = f"s3://{S3_BUCKET}/{key}"
     logger = prefect.context.get("logger")
     logger.info(f"Snowflake loading key: {uri}")
-    return SnowflakeQuery(**SNOWFLAKE_DEFAULT_DICT).run(data={'uri': uri}, query=IMPORT_SQL)
+    return SnowflakeQuery(**SNOWFLAKE_DEFAULT_DICT).run(
+        data={"uri": uri}, query=IMPORT_SQL
+    )
 
 
 @task()
 def cleanup(key: str):
     bucket = S3_BUCKET
     logger = prefect.context.get("logger")
     logger.info(f"deleting file: {str(key)}")
-    s3 = boto3.resource('s3')
+    s3 = boto3.resource("s3")
     s3.Object(bucket, key).delete()
 
 
-with Flow(FLOW_NAME, executor=LocalDaskExecutor(), schedule=get_interval_schedule(minutes=1440)) as flow:
+@task()
+def ordered_dataset_insert_transform() -> None:
+    """This task will incrementally insert new records into the article_content_ordered_live table.
+    This table was created manually from existing raw data as a starting point.
+    Query is built so that this task is itempotent.
+    """
+    logger = prefect.context.get("logger")
+    logger.info("Adding new records to article_content_ordered_live dataset...")
+    with sf.connect(**SNOWFLAKE_DEFAULT_DICT) as conn:
+        result = conn.execute_string(INSERT_ORDERED_SQL)
+        logger.info(f"Inserted {result[1].rowcount} rows...")
```

Comment:
Just curious: Why did you choose to do this in Prefect instead of Dbt?

Reply:
I wanted this work to be considering a new raw dataset....think of it as the 'tiny t' in EtLT.  It seemed better to keep this closer to data extraction, so that it can process the batch of data as it comes in.  Also the weekly reorder would be hard to implement in dbt and I wanted to keep the logic together.